src/redux/Books/Books.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Books = () => (
+    <div>
+      <h1>All Books</h1>
+    </div>
+  )
+
+export default Books;
```

Comment:
- [ ] Let's add a form as requested in the project instructions. 

<img width="836" alt="Capture" src="https://user-images.githubusercontent.com/58504655/134935714-9f2cf3bd-24dd-4084-9158-28d0f4384567.PNG">